runtime/jvmJsTest/src/kotlinx/serialization/MapperTest.kt

```diff
@@ -30,8 +30,7 @@ class MapperTest {
     data class NullableData(val nullable: String?, val nullable2: String?, val property: String)
```

Comment:
`MapperTest` does not pass, but it's not the issue right now.
Let's focus on the general concept and plugin changes, eventually, I will fix it

Reply:
I believe we need this test working so we can test that compiler still generates valid code in case `Decoder` supports `decodeSequentially`.

However, `Mapper` is not very good case, because, as you may remember, it should be rewritten to properties decoder which supports optional fields, hence we can't rely on `decodeSequentially`.

Reply:
I'll fix `TaggedTest`, leaving `Mapper` and `MapperTest` out of scope.